vignettes/Spectra.Rmd

```diff
@@ -415,9 +425,48 @@ peaks data (m/z and intensity values) within each spectrum are:
 - `reduceSpectra`: filters individual spectra keeping only the largest peak for
   groups of peaks with similar m/z values.
 
-In the example below we select all spectra measured in the second mzML file and
-subsequently filter them to retain spectra measured between 175 and 189 seconds
-in the measurement run.
+In this example, we use the filterValues function to retain spectra with a base
+peak m/z close to 100 (+/- 30 ppm) and a retention time around 230 (+/- 5 s).
+
+```{r}
+sps_sub <- filterValues(sps_sciex, spectraVariables = c("basePeakMZ", "rtime"),
```

Comment:
Nice example with the base peak m/z - was not even aware that we have that spectra variable - and we definitely don't have a filter function for that particular variable.

Only, please always add a whitespace after the `,` to make the code a little more readable.